cpp/unreal_plugins/UrdfBot/Source/UrdfBot/UrdfBotPawn.cpp

```diff
@@ -47,9 +47,31 @@ AUrdfBotPawn::AUrdfBotPawn(const FObjectInitializer& object_initializer) : APawn
 void AUrdfBotPawn::SetupPlayerInputComponent(class UInputComponent* input_component)
 {
     Super::SetupPlayerInputComponent(input_component);
+
+    UPlayerInput* player_input = GetWorld()->GetFirstPlayerController()->PlayerInput;
+    auto keyboard_actions = Config::get<std::map<std::string, std::map<std::string, std::map<std::string, float>>>>("URDFBOT.URDFBOT_PAWN.KEYBOARD_ACTIONS");
+    for (auto& keyboard_acton : keyboard_actions) {
+        InputAxisBinding input_axis_binding;
+        input_axis_binding.axis_ = FName(GetName() + "::" + keyboard_acton.first.c_str());
+        input_axis_binding.apply_action_ = keyboard_acton.second["APPLY_ACTION"];
```

Comment:
Always use .at() for retrieving an item from a map. Only ever use [] on the left side of an assignment.

Reply:
fixed

Reply:
Using .at() requires both `APPLY_ACTION` and `ADD_ACTION` show up in each keyboard action, even if it is empty. It is not a big issue and I just want to point it out.